ceph-lttng/README.md

```diff
@@ -0,0 +1,12 @@
+Builds an image with Ceph compiled with LTTnG. The image also has LTTnG tools
+for starting, stopping, and viewing the tracepoints.
+
+===================================================
+Quickstart
+===================================================
+
+Build Ceph with LTTnG and put it in an image:
+
+```bash
+CEPH_SRC=~/ceph ./build
+```
```

Comment:
Oh, got it. That makes more sense: you don't have to mess with the daemon script and leave the customization to whoever writes the builder.